drivers/ovsdriver.go

```diff
@@ -378,13 +433,30 @@ func (d *OvsDriver) CreateEndpoint(id string) error {
 		}
 	}
 
+	// Get OVS port name
+	ovsPortName := getOvsPortName(intfName, skipVethPair)
+
 	// Ask the switch to create the port
-	err = sw.CreatePort(intfName, cfgEp, pktTag, cfgNw.PktTag, skipVethPair)
+	err = sw.CreatePort(intfName, cfgEp, pktTag, cfgNw.PktTag, cfgEpGroup.Burst, skipVethPair, epgBandwidth)
 	if err != nil {
 		log.Errorf("Error creating port %s. Err: %v", intfName, err)
 		return err
 	}
 
+	// save local endpoint info
+	if epgKey != "" {
+		d.oper.LocalEpInfo[id] = &EpInfo{
```

Comment:
It'll be good to save endpoint info always.. If epgKey is empty, just save it as empty..
This way, we can reuse endpoint info for other purposes in future..